src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/categories.less

```diff
@@ -1,26 +1,26 @@
 @list-categories-gap: 5px;
 
-@list-categories-item-min-height: 85px;
+@list-categories-item-min-height: 95px;
 @list-categories-item-padding: 10px;
-@list-categories-item-image-width: 50px;
+@list-categories-item-image-width: 90px;
 
 .list-categories {
     .reset-ul();
+
     margin-bottom: @margin-gap;
 
-    @media @query-sm {
-        margin-left: -@list-categories-gap;
+    .wrap-border;
```

Comment:
I understand ... but maybe here is desirable to change it on every place because it's fragment of desing .. so for consistency is it here. Do you agree?